mlir/lib/Target/LLVMIR/Dialect/GENX/GENXToLLVMIRTranslation.cpp

```diff
@@ -202,6 +203,16 @@ static llvm::Value *createAtomicRMW(llvm::IRBuilderBase &builder,
                                   {ptr->getType(), val->getType()}, {ptr, val});
 }
 
+// Create call to a GenISAIntrinsic function
+static llvm::Value *createGenISAIntrinsicCall(
+    llvm::IRBuilderBase &builder, llvm::GenISAIntrinsic::ID intrin_id, ArrayRef<llvm::Value *> args, 
+    ArrayRef<llvm::Type *> tys) {
```

Comment:
Can we get `tys` from `args` instead of passing it?

Reply:
The 'tys' here is the return value type, so we have to pass it in.

Reply:
oic, can we rename it to `retType`?

Reply:
Done.